ai-catalog/gradle/libs.versions.toml

```diff
@@ -76,6 +77,7 @@ ui-tooling = { group = "androidx.compose.ui", name = "ui-tooling", version.ref =
 google-firebase-ai = { group = "com.google.firebase", name = "firebase-ai", version.ref = "firebaseAi" }
 androidx-lifecycle-viewmodel-android = { group = "androidx.lifecycle", name = "lifecycle-viewmodel-android", version.ref = "lifecycleViewmodelAndroid" }
 material3 = { group = "androidx.compose.material3", name = "material3", version.ref = "material3" }
+androidx-runtime = { group = "androidx.compose.runtime", name = "runtime", version.ref = "runtime" }
```

Comment:
is there a reason you needed the runtime version separate? this should be part of the BOM